pages/news/gadgetbridge-support.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Gadgetbridge adds AsteroidOS support
+layout: news
+posted: 2023-03-03
+post: true
+thumbnails: gadgetbridge
+summary: Gadgetbridge has added support for all AsteroidOS watches in release 0.73.0. Thanks to Noodelz for this great work on a near feature-complete initial implementation!
+author: Timo Könnecke (eLtMosen)
+---
+<h1 class="news-header-centered">A new way to sync with your watch!</h1>
+<div class="news-two-column">
+<div class="news-two-col-paragraph"><a href="https://codeberg.org/Freeyourgadget/Gadgetbridge">Gadgetbridge</a>, the application that allows you to connect your Pebble, Mi Band, Amazfit Bip and HPlus device and many more devices to your Android phone, can now sync to all AsteroidOS watches since <a href="https://codeberg.org/Freeyourgadget/Gadgetbridge/src/branch/master/CHANGELOG.md">release of version 0.73.0</a></div>
```

Comment:
I think we can make this paragraph a tiny bit easier to read (and mention that Gadgetbridge is open-source) with something like

Gadgetbridge, the open-source Android application that supports connection to Pebble, Mi Band, Amazfit Bip devices and many devices can now also sync to AsteroidOS watches since v0.73.

Reply:
Sounds better indeed!